src/doc/trpl/operators-and-overloading.md

```diff
@@ -1,3 +1,85 @@
 % Operators and Overloading
 
-Coming soon!
+Rust does not have general support for overloading operators or creating new
+operators. However, there are certain operators in Rust that _are_ able to
+be overloaded. To support a particular operator between types, there’s a
+specific trait that you can implement, which then overloads the operator.
+
+For example, the `+` operator can be overloaded with the `Add` trait:
+
+```rust
+use std::ops::Add;
+
+#[derive(Debug)]
+struct Point {
+    x: i32,
+    y: i32,
+}
+
+impl Add for Point {
+    type Output = Point;
+
+    fn add(self, other: Point) -> Point {
+        Point { x: self.x + other.x, y: self.y + other.y }
+    }
+}
+
+fn main() {
+    let p1 = Point { x: 1, y: 0 };
+    let p2 = Point { x: 2, y: 3 };
+
+    let p3 = p1 + p2;
+
+    println!("{:?}", p3);
+}
+```
+
+In `main`, we can use `+` on our two `Point`s, since `we’ve implemented
+`Add<Output=Point>` for `Point`.
+
+All of these traits follow a similar pattern. Let’s look at [`Add`][] in more detail:
```

Comment:
The phrase "All of these traits" also seems a bit out of place as I don't think the others have been mentioned yet? It may also be nice to point out the `std::ops` module which should have all these traits.